tests/nvme_test.py

```diff
@@ -31,6 +31,7 @@
 import subprocess
 import sys
 import unittest
+from time import sleep
```

Comment:
`time.sleep` and `time.time` are being used

```suggestion
import time
```